docs/reference/docker_tips.md

```diff
@@ -0,0 +1,42 @@
+# Docker Tips
+
+## Using Nvidia GPU Cloud (NGC)
+
+The standard JetBot containers are hosted on docker hub, but for development or testing purposes, you may wish to use Nvidia GPU Cloud. To access containers on NGC, you need to sign into the NGC registry. To do this:
+
+1. Log into ngc.nvidia.com
+
+2. Generated your API Key
+
+    1. Navigate to Setup > API Key
+    2. Click on "Generate API Key" at the right upper corner of the page
+    3. Copy the API key for future use
+
+3. On your Jetson, sign into the NGC registry using the generated key
+
+    ```bash
+    $ sudo docker login nvcr.io
+    
+    Username: $oauthtoken
+    Password: <Your Generated Key>
+    ````
+
+For the username, enter `$oauthtoken` exactly as shown. It is a special authentication token for all users.
+
+
+## Using custome Jupyter Lab workspace directory
+
+If you do work outside the /workspace directory, it will be lost when the container shuts down.
```

Comment:
There was more information related to workspace directory that was mentioned in previous review comment that should be included here.